### ISSUE TITLE
Modify the Procfile

### ISSUE THREAD

**Issue by BrayanCaro** [rising3/semver-gradle-plugin#6]
### Problem :cry: 

I think we can modify a `Procfile`.

This is a small file used by heroku to deploy an app, so if my jar filename is `app-v1.2.3.jar`, a `Procfile` will looks like this:

    web: java -jar build/libs/app-1.2.3.jar

You plugin is amazing and always work but I have to update this file and create a new commit manually on every release.

### Suggestion :eyes: 

I propose a flag like the following

    gradle semver --major --procfile

And the `Procfile` could be updated automatically

This works as an new feature without change the plugin behavior for the users who don't use heroku.

Or maybe using `semver extension` with

```gradle
semver {
    // To change the heroku Procfile (default: false):
    modifyProfile = true
}
```

**Comment by rising3** [comment]
Hi, thank you for using my plugin.

As for your problem, I had created a task in AWS Elastic beanstalk to automatically generate a `Procfile` with gradle.

For example, the following task.

``` gradle
task createProcfile {
    file("${projectDir}/Procfile").write("web: java -jar build/libs/${jar.archiveName}", "UTF-8")
}

bootJar.dependsOn createProcfile
```
Furthermore, I added the `Procfile` to .gitignore. This is my preference.

I hope this is helpful :smiley: 

Finally, I released a new version today, which uses Git tag names for version control.

Thank you.
